studioGhibliApi.js

```diff
@@ -0,0 +1,103 @@
+let container;
+let select;
+let opt;
+let ghibliArray;
+
+
+document.addEventListener('DOMContentLoaded', () => {
+    getMovieTitle()
+    let form = document.querySelector('form')
+    form.addEventListener('submit', (event) => {
+        submitReview(event);
+    })
+})
+const getApiData = async () => {
+    const movies = await axios.get('https://ghibliapi.herokuapp.com/films')
+    console.log(movies.data);
+    return movies.data
+}
+
+const submitReview = (event) => {
+    event.preventDefault()
+    opt = getSelectedMovie(select).text.bold()
+    let list = document.querySelector('ul')
+    let noInput = document.querySelector('#error')
+    let textInput = document.querySelector('#textField').value
+    let newReview = document.createElement('li')
+    if (textInput === '') {
+        noInput.innerText = `Please enter a valid review`
+        newReview.innerText = ''
+    } else {
+        noInput.innerText = ''
+        newReview.innerHTML = ` ${opt}: ${textInput}`
+        list.appendChild(newReview)
+    }
+    document.querySelector('#textField').value = ''
+}
+
+const pullFromHtml = async () => {
```

Comment:
Just like `getContainer()` this function didn't have to be an async function since grabbing elements from the DOM is a synchronous operation. This will also remove the need for `await` on `studioGhibliApi:43`